libraries/src/Categories/CategoryServiceTrait.php

```diff
@@ -45,7 +54,17 @@ trait CategoryServiceTrait
      */
     public function getCategory(array $options = [], $section = ''): CategoryInterface
     {
-        return $this->categoryFactory->createCategory($options, $section);
+        $options['access']      = (bool)($options['access'] ?? true);
+        $options['published']   = (bool)($options['published'] ?? true);
+        $options['countItems']  = (bool)($options['countItems'] ?? false);
+        
+        $hash = md5(serialize($options) . $section);
```

Comment:
```suggestion
        $options['countItems']  = (bool)($options['countItems'] ?? false);
        
        $hash = md5(serialize($options) . $section);
```
```suggestion
        $options['countItems']  = (bool)($options['countItems'] ?? false);

        $hash = md5(serialize($options) . $section);
```
Always the same issue with editors using automatic indentation.

Reply:
This should fix PHPCS failing.